### DISCUSSION TITLE
Test Data : Device Status

### DISCUSSION THREAD

**Discussion by psidana1983** [camaraproject/DeviceStatus#114] [Q&A]
Hi Team , 
I am working on a Device Status API and is looking for your support to suggest the test data for below case . Mentioned below is the statement from documentation for response field - "countryName" for case of Multiple Countries. 

" If there is mapping of one MCC to multiple countries, then we have list of countries. If there is no mapping of MCC to any country, then an empty array [] shall be returned.."

![image](https://github.com/camaraproject/DeviceStatus/assets/132988151/b9e8e0a6-d971-40b2-ada6-db26a16dc2f7)

Please help me with the scenario and test data for which the mentioned response is possible 

Regards
Prashant sidana

**Comment by rartych** [comment]
Here is the example:
https://github.com/camaraproject/DeviceStatus/pull/21#discussion_r1099934145
According to this site https://mcc-mnc.com/ MCC: **647** is still valid example.
